ua-frontend/.angular-cli.json

```diff
@@ -25,12 +25,29 @@
     "schematics": {
       "collection": "@nrwl/schematics",
       "postGenerate": "npm run format",
-      "newProject": ["app", "lib"]
+      "newProject": [
+        "app",
+        "lib"
+      ]
     },
     "styleExt": "scss",
     "component": {}
   },
   "warnings": {
     "typescriptMismatch": false
-  }
+  },
+  "apps": [
+    {
+      "name": "ua-common",
```

Comment:
I have always found "common" and "core" really unhelpful. That said, I end up using those names anyways . because it is hard to do anything differently. However, we should at least give this some thought. Typically these types of libraries are used to hold shared code, but because they are called common/core, there is not specific purpose so it ends up being a dumping ground of any and all shared code.

An alternative to this is to think through the different types of shared code we may need and create micro libraries that are specifically targeted to one purpose. Going down this path usually results in a proliferation of many small libraries, but with the NX setup perhaps that isn't that bad.

Also, I wonder if we should namespace with the understanding of a higher level project designation (i.e. ua-frontend vs ua-backend) and then one level down which is more granular (i.e. libs you have here which are currently called ua-common and ua-core). In thinking about this, is ua-core intended to be shared by anything (including the backend) or is it just shared among the front end?